MaxText/checkpointing.py

```diff
@@ -56,9 +56,8 @@ def create_orbax_checkpoint_manager(
 
 
 def load_state_if_possible(checkpoint_manager: CheckpointManager,
```

Comment:
It will be cleaner to split this function  - read from checkpoint_manager or read from a path. Also do not like the name if_possible. Should be handled cleanly by caller code.

Reply:
I think if_possible is a pretty common convention! There are lots of call sites so this gives clear semantics. Either way, I don't think you're commenting on the CR here?

Reply:
Yes - not related to this particular CR - but overall code.